### ISSUE TITLE
later imported prisma client class will emit logs unexpectedly when using multiple clients

### ISSUE THREAD

**Issue by EqualMa** [prisma/prisma#2812]
## Bug description

When using multiple prisma clients as described in [this comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118),
```typescript
import { PrismaClient as PrismaClient1 } from '../prisma/client1'
import { PrismaClient as PrismaClient2 } from '../prisma/client2'

const client1 = new PrismaClient1()
const client2 = new PrismaClient2()
```

it will output lots of logs when
creating new instance for the later imported client class.

For example, when only creating new instance, the following logs are output:

```
  prisma-client Client Version 2.0.0 +0ms
  prisma-client Engine Version de2bc1cbdb5561ad73d2f08463fa2eec48993f56 +2ms
  prisma-client {
  prisma-client   engineConfig: {
  prisma-client     cwd: '/workspace/prisma-multi-conn/prisma',
  prisma-client     debug: false,
  prisma-client     datamodelPath: '/workspace/prisma-multi-conn/generated/client2/schema.prisma',
  prisma-client     prismaPath: undefined,
  prisma-client     generator: {
  prisma-client       name: 'client',
  prisma-client       provider: 'prisma-client-js',
  prisma-client       output: '/workspace/prisma-multi-conn/generated/client2',
  prisma-client       binaryTargets: [],
  prisma-client       config: {},
  prisma-client       isCustomOutput: true
  prisma-client     },
  prisma-client     showColors: false,
  prisma-client     logLevel: undefined,
  prisma-client     logQueries: undefined,
  prisma-client     flags: [],
  prisma-client     clientVersion: '2.0.0'
  prisma-client   }
  prisma-client } +0ms
```

## How to reproduce

clone this repo: https://github.com/EqualMa/prisma-multi-conn
and then:
```
npm install
npm run gen 
npx ts-node script.ts
```

*DO NOT* specify `DEBUG="*"` because with this env var all clients will emit logs as expected.

## Expected behavior

No logs should be emitted

## Prisma information

can be seen in the repo.

## Environment & setup

- OS: Ubuntu 20.04
- Database: MySQL (my own project where I found this issue) and SQLite (the repro repo)
- Prisma version: 2.0.0
- Node.js version: v12.18.1

**Comment by beeplin** [comment]
got the same issue in postgres

**Comment by pantharshit00** [comment]
Looks like this is related to #2787 and might be closed a a duplicate. 

#2787 is fixed in dev and will be released in `2.1.0` tomorrow. 

After upgrading to dev, I can't reproduce this:
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/22195362/85285259-371a4880-b4ae-11ea-8fc1-db9f6b0bb6d2.png">

To install the dev version now, run:
```
npm install @prisma/cli@dev @prisma/client@dev
```

After that regenerate the client.

**Comment by pantharshit00** [comment]
I am actually going to close this as a duplicate of #2787. Feel feel to comment here if above doesn't fix the issue.
